core/src/lib.rs

```diff
@@ -12,11 +14,11 @@ pub trait DIDResolver {
     // Returns the DID Method that the DID Resolver is compatible with. Each resolver can only be compatible with one.
     fn get_method() -> &'static str;
     // Given a `did` and `key` it will construct the proper `verificationMethod` to use as part of the data integrity proof creation process.
-    fn create_verification_method(did: String, key_id: String) -> String {
+    fn create_verification_method(public_key: String, key_id: String) -> String {
         return format!(
             "did:{}:{}#{}",
```

Comment:
These two variables _should_ be able to be templated:
```suggestion
            "did:{}:{public_key}#{key_id}",
```